test/testlib/minikube_utilities.go

```diff
@@ -244,6 +244,26 @@ func InjectTestVersion(t *testing.T, options *helm.Options) {
 	options.SetValues["nuodb.image.tag"] = image.Nuodb.Image.Tag
 }
 
+func OverrideUpgradeRepository(t *testing.T, options *helm.Options) {
```

Comment:
The method says that it overwrites the repository but it actually overwrites also the tag unconditionally.
What if a specific test needs to test upgrade from NuoDB `4.0.8` instead of `4.0.4`?

Another option would be to re-map the tag, if set by the test, in case we are running internally. I.e `4.0.4` -> `rel-4.0.4-ce-centos7`.

Reply:
the scheme is different. The tag is 2. The repo is rel/4.0.4... It does not match the docker naming scheme